Xappy/Xappy/Content/Settings/StyleGuidePage.xaml.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Xappy.Content.Settings
 {
-    [XamlCompilation(XamlCompilationOptions.Skip)]
+    [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class StyleGuidePage : ContentPage
```

Comment:
@davidortinau I set this back to Compile.. It looks like whatever was breaking this no longer is and this will cause the UISegmentedControl to not get linked out